src/simulations/initial_conditions.jl

```diff
@@ -338,24 +437,38 @@ The returned function is a closure for `ic_path`. It could also be for `model`,
 many other ClimaLand functions are, but we wish to preserve the argument `model`
 in `set_ic!` for users who wish to define their own initial condition function,
 which may require parameters, etc, stored in `model`.
+
+If `enforce_constraints = true`, we ensure the soil water content is between porosity and the
+residual value, and that the temperature is bounded to be within the extrema of the air temperature
+at the surface.
+
+It is assumed that in CoupledAtmosphere simulations that `p.drivers.T` has 
+been updated already.
```

Comment:
With the current initialization in the coupler this won't be the case - we initialize all models with some ICs, then perform an initial exchange (which is where p.drivers gets filled)